tests/unit/disco/components/TestFooter.js

```diff
@@ -0,0 +1,26 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { FooterBase } from 'disco/components/Footer';
+import { getFakeI18nInst } from 'tests/unit/helpers';
+
+
+function shallowRender(props) {
```

Comment:
I'm surprised this doesn't need `props = {}` since it's optional. I guess the JSX spread handles nulls.